src/extensions/cite/index.ts

```diff
@@ -10,23 +61,46 @@ import { ready, create, replace, select, attr } from '../../util'
  * </span>
  * ```
  *
- * So that we can style the date, a better encoding would simply be:
+ * A better encoding for styling, and concision, is:
  *
  * ```html
- * <time datetime="2019-08-23" itemscope="" itemtype="http://schema.org/Date" itemprop="datePublished" >2019-08-23</time>
+ * <time datetime="2019-08-23" itemprop="datePublished" >2019-08-23</time>
  * ```
```

Comment:
Yes, I found that error goes away when you remove the itemscope="" itemtype="http://schema.org/Date" - and GSDTT still parses the date correctly. I prefer the simplicity of that. 

I does mean that the Encode `html` codec needs to handle `<time>` elements when decoding, instead of relying on the `Date` `itemtype`. But I think that is OK.